### PR TITLE
Add loading screen

### PR DESCRIPTION
**Pull request by sarah-witt**

### CODE REVIEW

**Review by jrosen081 — COMMENTED** [review]
this looks super awesome. Can we put it over top of the create user screen and log in screen when signing in/creating user?

File: frontend/src/pages/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -576,12 +577,12 @@ const Dashboard: FunctionComponent<DashboardParams> = ({
       userSessionData.userId !== sessionParams.userId ||
       userSessionData.sessionId !== +sessionParams.sessionId)
   ) {
-    return <h1>Loading</h1>;
+    return <LoadingScreen></LoadingScreen>;
```
Comment (cindy1u0) [inline frontend/src/pages/Dashboard/Dashboard.tsx]: Not a big deal but we could just do `<LoadingScreen />` like you did in Quiz.tsx

**Review by cindy1u0 — COMMENTED** [review]
LGTM! Also second what Jack said

**Comment by sarah-witt** [comment]
> this looks super awesome. Can we put it over top of the create user screen and log in screen when signing in/creating user?

Where would this go? I might be missing something but right now I see a loading screen when creating a user or signing in (since creating a user/signing in redirects to the dashboard).

**Review by cindy1u0 — APPROVED** [review]
Oh wait ya I see it for signing it now

**Review by jrosen081 — APPROVED** [review]

**Pull request merged**